src/app/pages/entradas/ver-comprobante/ver-comprobante.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'ngx-ver-comprobante',
+  templateUrl: './ver-comprobante.component.html',
+  styleUrls: ['./ver-comprobante.component.scss'],
+})
+
+
+export class VerComprobanteComponent implements OnInit {
+  @Input() transaccion: any;
+  @Input() consecutivo: any;
+  @Input() descripcion: any;
+  totalCreditos: any;
+  totalDebitos: any;
+
+  constructor() { }
+
+  ngOnInit() {
+     this.totalCreditos = 0;
+     this.totalDebitos = 0;
+     this.transaccion.movimientos.forEach(obj => {
+                                              if (obj.TipoMovimientoId === 344) {
+                                                 this.totalDebitos += obj.Valor;
+                                              }
+                                              if (obj.TipoMovimientoId === 345) {
+                                                 this.totalCreditos += obj.Valor;
+                                              }
```

Comment:
Con lo comentado en la L2 de este archivo:
```suggestion
                                              if (obj.TipoMovimientoId === MVTO_DB) {
                                                 this.totalDebitos += obj.Valor;
                                              }
                                              if (obj.TipoMovimientoId === MVTO_CR) {
                                                 this.totalCreditos += obj.Valor;
                                              }
```